src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/service/ListingService.java

```diff
@@ -48,8 +65,8 @@ public List<Listing> getListings() {
 
 
 
-    public @NotNull Listing updateListing(Listing listingInput) {
-        Listing updatedListing = listingRepository.findById(listingInput.getId())
+    public @NotNull Listing updateListing(Listing listing, ListingPutDto listingInput) {
```

Comment:
we could do it like that but I think we should do it like in users where have the updated user as our argument with the id set and we just change the fields, which are set in the user arg. what do you think? because now we couldn't update internally (if, in which case however) without a request, without creating a dot object. which usually are not to be created by the dev